Common/include/config_structure.hpp

```diff
@@ -9199,15 +9221,55 @@ class CConfig {
   */
   string GetVolumeOutput_Field(unsigned short iField);
 
-  /*
+  /*!
   * \brief Get the convergence fields for monitoring
   * \param[in] iField - Index of the field
   * return Field name for monitoring convergence
   */
   string GetConv_Field(unsigned short iField);
   
-  /*
-  * \brief Get the number of convergence monitoring fields.
+
+  /*!
+   * \brief Get functional that is going to be used to evaluate the convergence of the windowed time average of the unsteady problem.
+   * \param[in] iField - Index of the field
+   * \return Field name for monitoring convergence
+   */
+  string GetWndConv_Field(unsigned short iField);
+
+  /*!
+   * \brief Get the number of iterations that are considered in the Cauchy convergence criteria for the windowed time average of the unsteady problem.
+   * \return Number of elements in the Cauchy criteria windowed time average of the unsteady problem.
+   */
+  unsigned short GetWnd_Cauchy_Elems(void);
+
+  /*!
+   * \brief Get the value of convergence criteria for the Cauchy method for the time averaged
+   *        windowed objective functions for unsteady flows
+   * \return Value of the convergence criteria.
+   */
+  su2double GetWnd_Cauchy_Eps(void);
+
+  /*!
+   * \brief Get the number of iterations that are not considered in the convergence criteria for the windowed average output function
+   * \return Number of iterations before starting with the convergence criteria for the windowed average output function.
+   */
+  unsigned long  GetWnd_StartConv_Iter(void);
+
+  /*!
+   * \brief Get the boolean value, whether the the Cauchy method for the time averaged
+   *        windowed objective functions for unsteady flows is used or not.
+   * \return Boolean value, if the criterion is used.
+   */
+  bool GetWnd_Cauchy_Crit(void);
+
+  /*!
+  * \brief Get the number of convergence monitoring fields for time convergence monitoring.
+  * return Number of convergence monitoring fields.
+  */
+  unsigned short GetnWndConv_Field();
```

Comment:
Thank you for making the changes, it seems all "Get" methods you added can be const (in general all get methods should be), please add the const qualifier to everything you can.

Reply:
Fixed it :)